apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -100,6 +101,16 @@ const providers: Provider[] = [
         }
       }
 
+      const limiter = rateLimit({
+        interval: 60 * 1000, // 1 minute
+        uniqueTokenPerInterval: 100, // Max 10 login retries per minute
```

Comment:
Why do we need `uniqueTokenPerInterval` here? That's a LRU specific thing for setting max cache size. This doesn't apply to `memory-cache`.

Reply:
Will fix, thanks for spotting it @zomars